database/migrations/2023_12_19_211314_user_unique_email.php

```diff
@@ -21,7 +30,7 @@ public function up(): void
     public function down(): void
     {
         Schema::table('user', function (Blueprint $table) {
-            $table->dropUnique('email');
+            $table->dropUnique(['email']);
```

Comment:
Is this change necessary?  I believe you only need to specify an array if you want to drop multiple constraints.

Reply:
It is, the existing migration fails when you try to roll it back.

See the code snippet here for more info:
https://laravel.com/docs/10.x/migrations#dropping-indexes

Reply:
Oh yeah, I see now.  Passing a string interprets it as the entire index name, while passing an array is interpreted as index names Laravel knows how to create.  We should probably be testing the down migrations more thoroughly.  I'll add that as a todo to my list of things to do once I have time available to work on CDash development again.